docs/diagrams/init.md

```diff
@@ -0,0 +1,32 @@
+title etcdadm init
+
+participant user
+participant remote repo
+participant etcdadm
+participant systemd
+database member1
+
+user->etcdadm: init
+
+group download (if not in cache)
+etcdadm->remote repo: download etcd, etcdctl into local cache
+end
+
+group install artifacts
+etcdadm->host1: copy etcd, etcdctl from local cache to install directory
```

Comment:
Two of the participants in this diagram (`etcadm`, `systemd`) are processes; `host1` is not. In order to make the participants more parallel/similar, maybe `host1` here should be a `database` participant or be called the `filesystem of host1`.

Reply:
Good idea. I'll use `filesystem` in the `host1` `participantgroup`.